### ISSUE TITLE
wrong balances/dates

### ISSUE THREAD

**Issue by sandrafish** [datamade/nmid#47]
NM's Future Leaders is shown with a $149.26 balance [here](https://nmid.datamade.us/committees/?page=10) but a zero balance [here](https://nmid.datamade.us/committees/nms-future-leaders-298/)
same with [this PAC](https://nmid.datamade.us/committees/new-mexicans-for-honest-courts-77/)
[PAC West](https://nmid.datamade.us/committees/pac-west-303/)
this [guy](https://nmid.datamade.us/committees/sam-bregman-for-democratic-state-chair-364/)
this [group]([url](https://nmid.datamade.us/committees/democratic-women-of-socorro-280/))

anyway if you want a longer list i can do that - i suspect it's something to do with substituting the total expenditures for closing balance when the latter is 0, though it doesn't appear to be the case with all the 0 balance committees.

**Comment by sandrafish** [comment]
also, wow these NEA things are so confusing... [this](https://nmid.datamade.us/committees/nea-new-mexico-advocacy-fund-for-public-education-336/) shows a $586K donation in 2016 that doesn't show up elsewhere in the page... the NEA stuff is so complicated...

**Comment by sandrafish** [comment]
and this[ union](https://nmid.datamade.us/committees/central-new-mexico-labor-council-47/) with a large contribution that appears not to be reflected

this other [NEA account](https://nmid.datamade.us/committees/national-education-association-new-mexico-40/) too i'd note that the $265K isn't listed on [their PDF ](https://www.cfis.state.nm.us/docs/FPReports/0_46647_2016_9_9_152022.pdf) either tho. And it doesn't show up [here](https://nmid.datamade.us/search/?term=NEA-NM%20Members).

And these [folks](https://nmid.datamade.us/committees/albuquerque-area-fire-fighters-iaff-local-244-pac-339/). 

there are other instances of this too, seems like it's a labor union issue.

**Comment by sandrafish** [comment]
OK, so maybe those are running totals now that i look at it closer - and maybe "latest donation date" instead of latest date...

but still sticking with the 0 balance issue

**Comment by evz** [comment]
@sandrafish Yup, those are cumulative totals. Sorry about the ambiguity. I'll fix the header on the table. I also figured out what was going on with the $0 balance stuff. Had to do with not excluding certain kinds of supplemental filings that don't really work the way the periodic filings work.
